GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -555,8 +555,9 @@ void FramebufferManagerCommon::SetDepthFrameBuffer(bool isClearingDepth) {
 	if (!isClearingDepth && useBufferedRendering_) {
 		CopyToDepthFromOverlappingFramebuffers(currentRenderVfb_);
 
-		// Special compatibility trick for Burnout Dominator lens flares. See issue #11100
-		if (PSP_CoreParameter().compat.flags().UploadDepthForCLUTTextures && currentRenderVfb_->z_address > 0x04110000) {
+		// Need to upload the first line of depth buffers, for Burnout Dominator lens flares. See issue #11100 and comments to #16081.
+		// Might make this more generic and upload the whole depth buffer if we find it's needed for something.
+		if (currentRenderVfb_->lastFrameNewSize == gpuStats.numFlips) {
```

Comment:
Hm, I suppose if we did a overlapping framebuffer copy we don't want this even on the first frame.  That said, probably an uncommon situation.  Might cause flickering in Jeanne d'Arc or something.

-[Unknown]

Reply:
Well, we're currently only copying one line of the depth buffer here, so the flicker would be small :P

Reply:
I did forget that though, doesn't seem like a big deal after all..

-[Unknown]